Modules/Core/Common/CMake/itkCheckHasFenvtStructMember.cxx

```diff
@@ -21,15 +21,14 @@
 int
 main()
 {
-  fenv_t fenv;
+  [[maybe_unused]] fenv_t fenv;
 #if defined(ITK_CHECK_FENV_T_CONTROL)
-  (void)sizeof(fenv.__control);
+  [[maybe_unused]] const auto tempSize = sizeof(fenv.__control);
```

Comment:
Basically this just tests that `fenv.__control` is there, right? Maybe then it would be nicer to make that a static_assert:

    static_assert( sizeof(fenv.__control) > 0 );

Just suggestion.

Reply:
@hjmjohnson Just curious, what did you think of my suggestion to do `static_assert( sizeof(fenv.__control) > 0 )`, instead of introducing an unused `tempSize` variable?